submission-history/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Start with a Node.js base image.
+FROM node:carbon
+
+# Install dependencies.
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install
+
```

Comment:
You need to copy the `views` directory into the Docker image.

Reply:
No longer working on views so I will leave it as it for now